manim/mobject/geometry.py

```diff
@@ -104,12 +103,17 @@ class TipableVMobject(VMobject):
 
     """
 
-    CONFIG = {
-        "tip_length": DEFAULT_ARROW_TIP_LENGTH,
-        # TODO
-        "normal_vector": OUT,
-        "tip_style": dict(),
-    }
+    def __init__(
+        self,
+        tip_length=DEFAULT_ARROW_TIP_LENGTH,
```

Comment:
Do we want to move parameters like tip_length somewhere else?
With this defined here, also Mobjects like Dot() will have parameters tip_length and tip_style.
I guess that this does not bother too much, but it also does not make that much sense

Reply:
Currently, you can also change these `TipableVMobject`-specific properties in a CONFIG dict of `Dot`, so I don't think that anything changed practically. With this branch, you can just set it as a keyword; that is new.

(It should be said that all of this `TipableVMobject` stuff is really messy and should be refactored generously; we could, for example, let `Arc` and `Line` simply inherit from `VMobject`, and then have `Arrow` inherit from both `Line` and a new `Tipable` class which enables all of the business with the tips.

But this definitely is a discussion for another PR. 😄

Reply:
yep :)

Reply:
The idea here is to move everything into the __init__ function as keyword arguments and then later, clean each one up in turn.. 
So the answer is: we want to do that in a subsequent PR once CONFIGs are gone.